cmplibrary/src/test/java/com/sourcepoint/cmplibrary/SourcePointClientBuilderTest.java

```diff
@@ -0,0 +1,72 @@
+package com.sourcepoint.cmplibrary;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import java.lang.reflect.Field;
+
+import static org.junit.Assert.*;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+
+public class SourcePointClientBuilderTest {
+    SourcePointClientBuilder sourcePointClientBuilder;
+
+    @Before
+    public void setUp() throws ConsentLibException {
+        sourcePointClientBuilder = new SourcePointClientBuilder(808, "siteName", true);//mock(SourcePointClientBuilder.class);
```

Comment:
What if instead of using an instance variable we turned the initialisation of the `SourcePointClientBuilder` into a function? Example: 
``` java
private SourcePointClientBuilder defaultClientBuilder() {
  return new SourcePointClientBuilder(808, "siteName", true);
}
```
This way we can call it any time in our tests cases, instead of having to use an assignment 
```java
sourcePointClientBuilder.setCmpDomain("cmpDomain");
...
SourcePointClientBuilder sourcePointClientBuilder1 = sourcePointClientBuilder;
```
We'll then:
```java
public void setMmsDomain() throws Exception {
        SourcePointClientBuilder clientBuilder = defaultClientBuilder()
        clientBuilder.setMmsDomain("mmsDomain");
        Field member = SourcePointClientBuilder.class.getDeclaredField("mmsDomain");
        member.setAccessible(true);

        assertEquals("mmsDomain", member.get(clientBuilder).toString());
    }
```

Reply:
will work on this.

Reply:
implemented as suggested.